src/m365/aad/commands/policy/policy-list.ts

```diff
@@ -0,0 +1,126 @@
+import { Logger } from '../../../../cli';
+import {
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import GraphCommand from '../../../base/GraphCommand';
+import commands from '../../commands';
+import request from '../../../../request';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  policyType?: string;
+}
+
+const policyEndPoints: any = {
+  activitybasedtimeout: "activityBasedTimeoutPolicies",
+  authorization: "authorizationPolicy",
+  claimsmapping: "claimsMappingPolicies",
+  homerealmdiscovery: "homeRealmDiscoveryPolicies",
+  identitysecuritydefaultsenforcement: "identitySecurityDefaultsEnforcementPolicy",
+  tokenissuance: "tokenIssuancePolicies",
+  tokenlifetime: "tokenLifetimePolicies"
+};
+
+class AadPolicyListCommand extends GraphCommand {
+  public get name(): string {
+    return commands.POLICY_LIST;
+  }
+
+  public get description(): string {
+    return 'Returns policies from Azure AD';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.policyType = args.options.policyType || 'all';
+    return telemetryProps;
+  }
+
+  public defaultProperties(): string[] | undefined {
+    return ['id', 'displayName', 'isOrganizationDefault'];
+  }
+
+  public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
+    const policyType: string = args.options.policyType ? args.options.policyType.toLowerCase() : 'all';
+
+    if (policyType && policyType !== "all") {
+      this.getPolicies(policyType).then((policies?: any): void => {
+        logger.log(policies);
+        cb();
+      }, (err: any): void => this.handleRejectedODataJsonPromise(err, logger, cb));
+    }
+    else {
+      const policyTypes: string[] = Object.keys(policyEndPoints);
+      Promise.all(policyTypes.map(policyType => this.getPolicies(policyType)))
+        .then((results) => {
+          let allPolicies: any = [];
+          results.forEach((policies: any) => {
+            allPolicies = allPolicies.concat(policies);
+          });
+          logger.log(allPolicies);
+          cb();
+        }).catch(err => {
+          this.handleRejectedODataJsonPromise(err, logger, cb)
+        });
+    }
+  }
+
+  private getPolicies(policyType: string): Promise<any> {
+    const endpoint = policyEndPoints[policyType];
+    const requestOptions: any = {
+      url: `${this.resource}/v1.0/policies/${endpoint}`,
+      headers: {
+        accept: 'application/json;odata.metadata=none'
+      },
+      responseType: 'json'
+    };
+
+    return request
+      .get<{ value: any[] }>(requestOptions)
+      .then((response: any) => {
```

Comment:
Since we're defining the response as `any` to support returning different types, there is no point in defining `{value:any[]}` type in the generic `get` as it's not used in any way.